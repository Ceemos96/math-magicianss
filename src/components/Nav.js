import React from 'react';
import { Link } from 'react-router-dom';

const Nav = () => (
  <div className="nav-wrapper">
    <Link className="heading-text" to="/">
      Math Magicians
    </Link>
    <ul className="nav-links">
      <li className="nav-link">
        <Link activeStyle={{ color: 'blue' }} to="/">
          Home
        </Link>
      </li>
      <li className="nav-link">
        <Link activeStyle={{ color: 'blue' }} to="/calculator">
          Calculator
        </Link>
      </li>
      <li className="nav-link">
        <Link activeStyle={{ color: 'blue' }} to="/quotes">
          Quote
        </Link>
      </li>
    </ul>
  </div>
);
export default Nav;
