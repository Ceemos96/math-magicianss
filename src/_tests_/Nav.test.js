import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Nav from '../components/Nav';

describe('Testing the Navbar', () => {
  it('Nav is rendered', () => {
    const nav = renderer
      .create(<BrowserRouter><Nav /></BrowserRouter>)
      .toJSON();
    expect(nav).toMatchSnapshot();
  });
});
