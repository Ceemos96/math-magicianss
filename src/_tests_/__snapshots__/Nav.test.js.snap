// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Navbar Nav is rendered 1`] = `
<div
  className="nav-wrapper"
>
  <a
    className="heading-text"
    href="/"
    onClick={[Function]}
  >
    Math Magicians
  </a>
  <ul
    className="nav-links"
  >
    <li
      className="nav-link"
    >
      <a
        activeStyle={
          Object {
            "color": "blue",
          }
        }
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li
      className="nav-link"
    >
      <a
        activeStyle={
          Object {
            "color": "blue",
          }
        }
        href="/calculator"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li
      className="nav-link"
    >
      <a
        activeStyle={
          Object {
            "color": "blue",
          }
        }
        href="/quotes"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</div>
`;
